README.md

```diff
@@ -1,7 +1,7 @@
 MCPE Linux Launcher
 ===================
 
-## Required packages on Ubuntu
+## Setup for Ubuntu
```

Comment:
Since, you are modifying the README, you may change Ubuntu to Linux. Since, does are the requirement for any linux distro, not only for Ubuntu

Reply:
The instructions are Ubuntu-specific, so I'll leave it up to MrARM if they want changes to be made